environment/values.yaml

```diff
@@ -1,17 +1,33 @@
 labels: {}
 
+company:
+  name: #example
+  gcp:
+    enabled: false
+    bucket:
+      prefix:
+    project_id: #example-production
+  domain:
+    root: #example.com
+    env: #int.example.com, dev.example.com, prd.example.com etc
+  network:
+    int_nat_gw:
```

Comment:
```
  network:
    int_nat_gw:
```
and here?

Reply:
same as here https://github.com/ggramal/helm-charts/pull/8#discussion_r1613354466